# HG changeset patch
# User Andrew John Hughes <ahughes@redhat.com>
# Date 1317353129 -3600
# Node ID 7cdc3acdda5b3eab0535c59cd348966a5b825d96
# Parent  f75092cac18973ae8c71794bcdd8d0fd71804f4c
Use IcedTea server for CACAO downloads, as with 6.

2010-11-19  Andrew John Hughes  <ahughes@redhat.com>

	* Makefile.am:
	(CACAO_BASE_URL): Switch to using IcedTea
	server for CACAO drops too.
	(CACAO_URL): Likewise.

diff -r f75092cac189 -r 7cdc3acdda5b ChangeLog
--- a/ChangeLog	Fri Sep 30 04:07:32 2011 +0100
+++ b/ChangeLog	Fri Sep 30 04:25:29 2011 +0100
@@ -1,3 +1,10 @@
+2010-11-19  Andrew John Hughes  <ahughes@redhat.com>
+
+	* Makefile.am:
+	(CACAO_BASE_URL): Switch to using IcedTea
+	server for CACAO drops too.
+	(CACAO_URL): Likewise.
+
 2011-09-30  Andrew John Hughes  <ahughes@redhat.com>
 
 	Bump changesets to bring in fix which
diff -r f75092cac189 -r 7cdc3acdda5b Makefile.am
--- a/Makefile.am	Fri Sep 30 04:07:32 2011 +0100
+++ b/Makefile.am	Fri Sep 30 04:25:29 2011 +0100
@@ -20,8 +20,8 @@
 
 CACAO_VERSION = d6264eb66506
 CACAO_SHA256SUM = 94ea7899e806ccbc33a732b5113a8f969d8b1f4ce7ffd27cf04577054f65f63c
-CACAO_BASE_URL = http://mips.complang.tuwien.ac.at/hg/cacao/archive
-CACAO_URL = $(CACAO_BASE_URL)/$(CACAO_VERSION).tar.bz2
+CACAO_BASE_URL = http://icedtea.classpath.org/download/drops/cacao
+CACAO_URL = $(CACAO_BASE_URL)/$(CACAO_VERSION).tar.gz
 CACAO_SRC_ZIP = cacao-$(CACAO_VERSION).tar.bz2
 
 JAMVM_VERSION = 310c491ddc14e92a6ffff27030a1a1821e6395a8
